src/main/java/frc/robot/RobotContainer.java

```diff
@@ -40,16 +64,36 @@ public RobotContainer() {
     // load paths before auto starts
     PathGroupLoader.loadPathGroups();
 
-    Driver.configureControls(m_drive);
-    Operator.configureControls(m_arm);
+    Driver.configureControls(m_drive, m_arm);
+    Operator.configureControls(m_drive, m_arm);
     TestControls.configureControls(m_drive);
 
+    Vision.setup(m_drive);
+
+    // Puts April tags in array
+    for(int i = 1; i <= 8; i++){
+      aprilTags[i] = Vision.getTagPose(i);
+    }
```

Comment:
Now the values are changed to actual Pose3D values, but why is aprilTags needed when there is Vision.getTagPose(i)?

Reply:
Read previous comment.